cpp/cmake/Version.cmake

```diff
@@ -1,7 +1,7 @@
 # #%L
 # Bio-Formats C++ libraries (cmake build infrastructure)
 # %%
-# Copyright © 2006 - 2016 Open Microscopy Environment:
+# Copyright © 2006 - 2015 Open Microscopy Environment:
```

Comment:
Note it's just a copy of what was in the Version.cmake in all the other repos--it needs updating across the board.